src/components/dashboard/Claim.js

```diff
@@ -44,7 +44,8 @@ import { BigGoodDollar, Section, WrapperClaim } from '../common/'
 import useAppState from '../../lib/hooks/useAppState'
 import { WavesBox } from '../common/view/WavesBox'
 import useTimer from '../../lib/hooks/useTimer'
-import useInterval from '../../lib/hooks/useInterval'
+
+// import useInterval from '../../lib/hooks/useInterval'
```

Comment:
```suggestion
import useInterval from '../../lib/hooks/useInterval'
```

Reply:
as you're not removing it totally, let it be configured with some env var - it would be easier to bring it back if something will go wrong